### PR TITLE
Mount status and multiple connect support

### PR DESCRIPTION
**Pull request by kukkjanos**
Support check mount status and multiple connect with comma separated number

### CODE REVIEW

File: sshfs-manager.sh [hunk 1]
```diff
@@ -22,8 +22,9 @@ showMessage() {
             [INVALID_COMMAND]='[SSHFS-MGR] Please run one of the available commands from the help menu (%s --help).\n'
             [NO_SERVERS]='[SSHFS-MGR] No servers were found. Add a server using the "add-server" command.\n'
             [AVAILABLE_SERVERS]='[SSHFS-MGR] Available servers:\n'
-            [USER_INPUT_SERVER_INDEX]='\n[SSHFS-MGR] Enter a number from the list: '
+            [USER_INPUT_SERVER_INDEX]='\n[SSHFS-MGR] Enter a number from the list (supported comma separated numbers): '
             [USER_INPUT_INVALID_SERVER_INDEX]='[SSHFS-MGR] Please select a valid number from the list. Exiting...\n'
+            [USER_INPUT_ILLEGAL_SERVER_INDEX]='[SSHFS-MGR] Please select a valid comma separated number number(s) from the list. Exiting...\n'
```
Comment (tanasebutcaru) [inline sshfs-manager.sh]: `USER_INPUT_ILLEGAL_SERVER_INDEX` is not used anywhere in this file, I think it should replace `USER_INPUT_INVALID_SERVER_INDEX`.

Also, what do you think if we'd change the message into: 
> Please enter a valid number or multiple comma separated numbers from the list. Existing...\n

File: sshfs-manager.sh [hunk 2]
```diff
@@ -268,21 +269,50 @@ connectPrompt() {
         serverDomain=${serverList[$index]}
         mountDir=${servers[$serverDomain,'mountDir']}
         domain=${serverList[$index]}
-        printf "\t$index - $mountDir @ $domain \n"
+	if mountpoint -q "${config[mountPath]}/$mountDir"; then
+	    mountPointStatus="(\e[1;92mOnline\e[0m)"
+	else
+	    mountPointStatus="(\e[1;91mOffline\e[0m)"
```
Comment (tanasebutcaru) [inline sshfs-manager.sh]: Let's add these two `mountPointStatus` into the `messages` dictionary.

**Review by tanasebutcaru — CHANGES_REQUESTED** [review]

**Review by kukkjanos — COMMENTED** [review]
I pushed a little fix!

**Comment by tanasebutcaru** [comment]
Looking good. Merged!
Thank you :+1:

**Pull request merged**